src/frontend/screens/Settings/MapSettings/BackgroundMapInfo.tsx

```diff
@@ -167,34 +165,38 @@ export const BackgroundMapInfo = ({
             allowUpdates={true}
           />
         </MapboxGL.MapView>
-        <ScrollView style={styles.container}>
-          {bytesStored && (
-            <Text style={{ color: MEDIUM_GREY }}>
-              {`${convertBytesToMb(bytesStored).toFixed(0)} ${t(m.mb)}`}
-            </Text>
-          )}
-
-          {id !== DEFAULT_MAP_ID && (
+        <View style={[styles.flex, styles.container]}>
+          <View>
+            {bytesStored ? (
+              <Text style={styles.detailsText}>
+                {`${bytesToMegabytes(bytesStored).toFixed(0)} ${t(m.mb)}`}
+              </Text>
```

Comment:
not sure if the proposed changes are necessary for this component because I think it makes sense to show the storage amount if it's a non-zero number, regardless of the type of style/map it is. it's possible that in the future, the custom or default map will have a definitive number (and not just explicitly passed `0`), which this will already handle.

Reply:
ah i see. i thought this was the BackgrdoundMaps screen, not the info screen for a background map. Updating based on your suggestion!